packages/flame/lib/src/components/scroll_text_box_component.dart

```diff
@@ -15,13 +16,17 @@ import 'package:flutter/painting.dart';
 /// capabilities.
 class ScrollTextBoxComponent<T extends TextRenderer> extends PositionComponent {
   late final _ScrollTextBoxComponent<T> _scrollTextBoxComponent;
+  late final ValueNotifier<int> _newLineNotifier;
+
+  ValueNotifier<int> get newLineNotifier => _newLineNotifier;
```

Comment:
```suggestion
  late final ValueNotifier<int> newLineNotifier;
```
It should be fine to have just this one

Reply:
Thanks, you're right :)